modules/cnf-topology-aware-lifecycle-manager-precache-user-spec-images.adoc

```diff
@@ -143,7 +143,19 @@ All sites are pre-cached concurrently.
 +
 [source,terminal]
 ----
-$ oc get cgu <cgu-name> -n <cgu-namespace> -oyaml
+$ oc get cgu <cgu_name> -n <cgu_namespace> -oyaml
+----
+
+.Example output
+[source,yaml]
+----
+status:
+  …
```

Comment:
The most important output to verify pertains to the `status.precaching` object, which should look something like this (I have modified a typical output to match the specifications in `PreCachingConfig CR` as well the `ClusterGroupUpgrade CR`):
```yaml
  precaching:
    spec:
      platformImage: quay.io/openshift-release-dev/ocp-release@sha256:3d5800990dee7cd4727d3fe238a97e2d2976d3808fc925ada29c559a47e2e1ef
      operatorsIndexes:
        - registry.example.com:5000/custom-redhat-operators:1.0.0
      operatorsPackagesAndChannels:
        - local-storage-operator: stable
        - ptp-operator: stable
        - sriov-network-operator: stable
      excludePrecachePatterns:
        - aws
        - vsphere
      additionalImages:
        - quay.io/exampleconfig/application1@sha256:3d5800990dee7cd4727d3fe238a97e2d2976d3808fc925ada29c559a47e2e1ef
        - quay.io/exampleconfig/application2@sha256:3d5800123dee7cd4727d3fe238a97e2d2976d3808fc925ada29c559a47adfaef
        - quay.io/exampleconfig/applicationN@sha256:4fe1334adfafadsf987123adfffdaf1243340adfafdedga0991234afdadfsa09
      spaceRequired: "30"
    status:
      sno1: Starting
      sno2: Starting
```